deepchem/metrics/score_function.py

```diff
@@ -162,3 +162,55 @@ def bedroc_score(y_true: np.ndarray, y_pred: np.ndarray, alpha: float = 20.0):
   scores = sorted(scores, key=lambda pair: pair[1], reverse=True)
 
   return CalcBEDROC(scores, 0, alpha)
+
+
+def concordance_index(y_true: np.ndarray, y_pred: np.ndarray) -> float:
```

Comment:
Can you add a autodoc for this new metric to `docs/metrics.rst` along with the other metrics there? That will ensure `concordance_index` shows up in our documentation